### PR TITLE
Adding Office 365 Script

### PR DESCRIPTION
**Pull request by mattco593**
Adding script that limits the users that are allowed to create Microsoft/Office 365 Groups to a specific group with the exceptions of certain administrative groups that are listed on the README.

### CODE REVIEW

File: scripts/powershell/Office-365/GroupCreators.ps1 [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿$GroupName = "Office 365 Group Creators"
```
Comment (hbwheat) [inline scripts/powershell/Office-365/GroupCreators.ps1]: @mattco593 , can you add a header here?
For example: 
```
################################################
#
# Author: 
# Contact: github link or email if you want
#
# Title:
# Purpose:
#
#
################################################
```

File: scripts/powershell/readme.md [hunk 2]
```diff
@@ -9,3 +9,7 @@ List of great repositories that are maintained and owned elsewhere.
 ### Cyber
 
 Cybersecurity powershell things...
+
+### Office-365
+
+Office 365 related scripts...
```
Comment (hbwheat) [inline scripts/powershell/readme.md:15]: @mattco593 , I edited the description to be more general as it won't just have Gov scripts but also Commercial.

File: scripts/powershell/Office-365/GroupCreators.ps1 (hunk 1 above)
Comment (mattco593) [reply scripts/powershell/Office-365/GroupCreators.ps1]: Yeah I will take care of that now.

File: scripts/powershell/Office-365/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Office 365 Scripts
```
Comment (hbwheat) [inline scripts/powershell/Office-365/README.md]: @mattco593 , let's put this closer to the script, make a new "group-creators" folder under Office-365? Or add this as a block comment?

**Review by hbwheat — CHANGES_REQUESTED** [review]
@mattco593 , see comments on files.

File: scripts/powershell/Office-365/GroupCreators.ps1 (hunk 1 above)
Comment (hbwheat) [reply scripts/powershell/Office-365/GroupCreators.ps1]: cool cool, just something generic, but to also give attribution and credit. :-)

**Review by mattco593 — COMMENTED** [review]
Made requested changes

**Review by hbwheat — APPROVED** [review]
Looks good!

File: scripts/powershell/Office-365/GroupCreators.ps1 (hunk 1 above)
Comment (mattco593) [reply scripts/powershell/Office-365/GroupCreators.ps1]: Done

**Pull request merged**